courts.md

```diff
@@ -5,112 +5,112 @@
 
 ## supreme_court
 
-| Name | Code | Param | Start | End | List? | Searchable? |
-| ---- | ---- | ----- | ----- | --- | ----- | ----------- |
-| United Kingdom Supreme Court | UKSC | uksc | 2009 | – | ✅ | ✅ |
+| Name | Code | Param | Example NCNs | Start | End | List? | Searchable? |
+| ---- | ---- | ----- | ------------ | ----- | --- | ----- | ----------- |
+| United Kingdom Supreme Court | UKSC | uksc | [2013]&nbsp;UKSC&nbsp;42 | 2009 | – | ✅ | ✅ |
```

Comment:
I don't love the `&nbsp;` since it makes it harder to read as markdown. Does it look rubbish without?